packages/react-error-overlay/webpack.config.iframe.js

```diff
@@ -43,6 +45,24 @@ module.exports = {
       },
     ],
   },
+  optimization: {
+    minimizer: [
+      // This code is embedded as a string, so it would never be optimized
+      // elsewhere.
+      new UglifyJsPlugin({
+        uglifyOptions: {
+          compress: {
+            warnings: false,
+            comparisons: false,
+          },
+          output: {
+            comments: false,
+            ascii_only: false,
+          },
+        },
+      }),
+    ],
+  },
   plugins: [
     new webpack.DefinePlugin({
```

Comment:
ditto: why reordered?

Reply:
using the new  optimization.minimizer  option in webpack 4